smoke/IotEdgeQuickstart/Program.cs

```diff
@@ -102,6 +102,9 @@ class Program
         [Option("--no-verify", CommandOptionType.NoValue, Description = "Don't verify the behavior of the deployment (e.g.: temp sensor)")]
         public bool NoVerify { get; } = false;
 
+        [Option("--bypass_edge_installation", CommandOptionType.NoValue, Description = "Don't install bootstrapper")]
```

Comment:
should be "--bypass-edge-installation"; use dash instead of underscore.

Reply:
Sorry for confusion; existing codes use dash and underscore; but the right one should use dash.

Reply:
did you test? for optimize for performance flag, existing code is like "--optimize_for_performance <true/false>"; it define with "<true/false>".  Please verify if this works.

Reply:
Renamed parameter to _--bypass-edge-installation_.
I tested the behavior of the newly introduced parameter _--bypass-edge-installation_ and it works as expected.
I have not tried the other parameter _optimize_for_performance <true/false>_ as I have not made any changes to that one.